src/screens/Nearby/section-items/line.tsx

```diff
@@ -72,29 +72,32 @@ export default function LineItem({
 
   const title = `${group.lineInfo?.lineNumber} ${group.lineInfo?.lineName}`;
 
-  const onPress = (departure: DepartureTime) => {
+  const items = group.departures.map((dep) => ({
+    serviceJourneyId: dep.serviceJourneyId!,
+    date: dep.time,
+    fromQuayId: group.lineInfo?.quayId,
+  }));
+
+  const onPress = (activeItemIndex: number) => {
     navigation.navigate('TripDetails', {
       screen: 'DepartureDetails',
       params: {
-        title,
-        serviceJourneyId: departure.serviceJourneyId!,
-        date: departure.aimedTime,
-        fromQuayId: group.lineInfo?.quayId,
+        activeItemIndex,
+        items,
```

Comment:
Was thinking about how we'd link to "trips" with deep-links when we get as far as providing that functionality. But we probably need some other way of looking up trips when we get that far anyway. Not that sending in `items` breaks anything, still serializable, just thinking out loud.

Reply:
I think this still supports deep linking in the same way as before. Items is just references to service journeys which can be URLs

Reply:
Yeah I think so too, just thinking out loud if this screen is maybe evolving past the scope of "linking to a single trip". If we ever get there that support can deep-link trips for users, or users can save trips. Not important wrt to this PR 👯

Reply:
Note, this is for service journeys, not trips.